dashboard/config/libraries/Outbreak.interpreted.js

```diff
@@ -3,6 +3,8 @@ var RECOVERY_TIME = 250;
 var RECOVERY_BAR_HEIGHT = 5;
 var PERCENT_SICK_AT_SETUP = 10;
 var SPRITE_SIZE = 25;
+var MIN_XY = 20;
+var MAX_XY = 380;
```

Comment:
should we add notes about these variables? i think MIN_XY is the min because of SPRITE_SIZE plus some padding (similar for MAX_XY), but a comment here might help, or we could use SPRITE_SIZE to "calculate" these constants to make that clearer